src/test/java/org/broadinstitute/hellbender/utils/io/IOUtilsUnitTest.java

```diff
@@ -205,6 +205,25 @@ public void testGetAbsolutePathWithoutFileProtocol(final String uriString, final
         Assert.assertEquals(IOUtils.getAbsolutePathWithoutFileProtocol(IOUtils.getPath(uriString)), expected);
     }
 
+    @Test
+    public void testAppendPathToDir() throws Exception {
+        Assert.assertEquals(IOUtils.appendPathToDir("dir", "file"), "dir/file");
+        Assert.assertEquals(IOUtils.appendPathToDir("dir/", "file"), "dir/file");
+        Assert.assertEquals(IOUtils.appendPathToDir("dir", "/file"), "/file");
+        Assert.assertEquals(IOUtils.appendPathToDir("dir/", "/file"), "/file");
+        Assert.assertEquals(IOUtils.appendPathToDir("/path/to/dir", "anotherdir/file"), "/path/to/dir/anotherdir/file");
+
+        // hdfs: URI
+        Path tempPath = IOUtils.getPath(MiniClusterUtils.getWorkingDir(MiniClusterUtils.getMiniCluster()).toUri().toString());
```

Comment:
Need to stop the cluster when done.

Actually there's no good reason to use a `MiniCluster` here -- just construct an appropriate String instead.